src/services/UsersService.ts

```diff
@@ -0,0 +1,4 @@
+import api from '../config/api';
+import { User } from '../utils/types';
```

Comment:
absolute paths

Reply:
Ok, I made the paths refactor to support absolute instead of relative paths.